apps/civicsignalblog/.eslintrc.js

```diff
@@ -1,11 +1,21 @@
 module.exports = {
   root: true,
   extends: ["eslint-config-commons-ui/next"],
+  rules: {
+    "react/jsx-filename-extension": [
+      "warn",
+      { extensions: [".js", ".jsx", "ts", ".tsx"] },
```

Comment:
Why? we shouldn't use `.jsx` and in TS, jsx is only allowed in `.tsx`.

Reply:
I think we need to completely remove this rule we do have a lot of jsx in js files and I believe there is no need to be too strict or treat them differently...I had introduced it as I was trying to resolve some linting issues on my editor, but I've just crosschecked it has nothing to do with this config  🫠